extensions/Khronos/KHR_materials_cmnBlinnPhong/README.md

```diff
@@ -0,0 +1,134 @@
+
+Draft (not ratified yet)
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines a Blinn-Phong model based material for use with glTF 2.0 in the same way as the core specification defines Physically Based Rendering (PBR) based materials. Other extension may define additional commonly used materials. For example, if a need for a Phong model arises, an extension named e.g. `cmnPhong` could be specified. 
+
+Given the long history of the Blinn-Phong model and the vast amount of collective experience with applying this model, the demand for convenient use of such materials is still there.
+
+This extension defines a single material type, Blinn-Phong. Other or simpler materials types such as Lambert or Constant can be described by having zero factors.
+
+## Extending Materials
+
+The common Blinn-Phong material is defined by adding the `KHR_materials_cmnBlinnPhong` extension to any glTF material. 
+For example, the following defines a material having all possible parameters.:
+
+```
+{
+    "materials": [
+        {
+            "name": "All parameters",
+            "extensions": {
+                "KHR_materials_cmnBlinnPhong": {
+                    "diffuseFactor" : [
+                        0.8, 
+                        0.8, 
+                        0.8, 
+                        1.0
+                    ], 
+                    "diffuseTexture" : {
+                        "index" : 0
+                    }, 
+                    "shininessFactor" : 2.0, 
+                    "specularFactor" : [
+                        0.1, 
+                        0.1, 
+                        0.1
+                    ],
+                    "specularShininessTexture" : {
+                        "index" : 1
+                    }, 
+                }
+            }
+
+        }
+    ]
+}
+```
+### Blinn - Phong
+
+The Blinn-Phong material model is defined by the following properties:
+* `diffuse` - Reflected diffuse color of the material
+* `specular` - Specular color of the material
+* `shininess` - Shininess factor of the specular color
+
+TODO: EXPLAIN FURTHER 
+
+The following table lists the allowed types and ranges for the specular-glossiness model:
+
+|                            |Type         |Description|Required|
+|----------------------------|-------------|-----------|--------|
+|**diffuseFactor**           | `number[4]` | The reflected diffuse RGBA factor of the material.|No, default: `[1.0,1.0,1.0,1.0]`|
+|**diffuseTexture**          | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | The diffuse texture.|No|
+|**specularFactor**          | `number[3]` | The specular RGB color of the material.      |No, default: `[1.0,1.0,1.0]`|
+|**shininessFactor**         | `number`    | The shininess of the material.|No, default: `1.0`          |
+|**specularShininessTexture**| [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)|The specular shininess texture.|No|

```

Comment:
I assume this means specular is `RGB` channels, and shininess is `A`. We've [discussed a bit in the past](https://github.com/KhronosGroup/glTF/issues/857#issuecomment-284258812), and I don't remember where we ended up on this question, but there was some concern that because using RGB+A requires PNG, transmission size might actually end up worse than if two JPGs had been used (e.g. RGB + R).